public/components/common/custom-search-bar/components/multi-select.tsx

```diff
@@ -28,7 +28,7 @@ import { IValueSuggestion, useValueSuggestion } from '../../hooks';
 const ON = 'on';
 const OFF = 'off';
 
-export const MultiSelect = ({ item, onChange, selectedOptions, onRemove }) => {
+export const MultiSelect = ({ item, onChange, selectedOptions, onRemove, isDisable }) => {
```

Comment:
Nitpick: gramatically is more correct to use `isDisabled` rather than `isDisable`